src/controllers/midi/legacymidicontrollermappingfilehandler.cpp

```diff
@@ -239,8 +239,13 @@ void LegacyMidiControllerMappingFileHandler::addControlsToDocument(
         for (auto it = mapping.getInputMappings().constFind(key);
                 it != mapping.getInputMappings().constEnd() && it.key() == key;
                 ++it) {
-            QDomElement controlNode = inputMappingToXML(doc, it.value());
-            controls.appendChild(controlNode);
+            std::visit(MidiUtils::overloaded{
+                               [&](ConfigKey control) {
+                                   QDomElement controlNode = inputMappingToXML(doc, it.value());
+                                   controls.appendChild(controlNode);
+                               },
+                               [](QJSValue control) { /* Do nothing */ }},
```

Comment:
As I understand here, this function writes an XML file from manually registered control in the Mixxx interface, so it should do nothing on a anonymous function from JS.

Reply:
Yup, sounds about right.